aliskin/assignment1/kth-element.py

```diff
@@ -0,0 +1,19 @@
+class ListNode:
+    def __init__(self, x):
+        self.value = x
+        self.next = None
+
+def kth_element(head, k):
```

Comment:
It's "kth_last_element", that is the kth element starting from the last one going backwards.